specs/sb/api.md

```diff
@@ -342,6 +400,45 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
         }, {
           "content": "40 concurrent connections"
         }]
+      },
+      "schemas": {
+        "service_instance": {
+          "parameters": {
+            "required": [
+              "fake_required_property"
+            ],
+            "properties": {
+              "fake_required_property": {
+                "type": "string"
+              }
+            }
+          },
+          "outputs": {
+            "properties": {
```

Comment:
There is an example of an output for both service instance and service binding in the example section. I have updated them to be more clear, please let me know if you still feel them lacking.